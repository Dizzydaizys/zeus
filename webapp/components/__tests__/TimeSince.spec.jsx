import React from 'react';
import {render} from 'enzyme';

import TimeSince from '../TimeSince';

describe('TimeSince', () => {
  it('renders default', () => {
    const tree = render(<TimeSince date="2017-10-15T01:41:20Z" />);
    expect(tree).toMatchSnapshot();
  });

  it('renders 24 hour clock', () => {
    const tree = render(<TimeSince date="2017-10-15T01:41:20Z" clock24Hours />);
    expect(tree).toMatchSnapshot();
  });
});
