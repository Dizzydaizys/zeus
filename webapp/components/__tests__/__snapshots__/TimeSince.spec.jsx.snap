// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimeSince renders 24 hour clock 1`] = `
<time
  datetime="2017-10-15T01:41:20.000Z"
  title="October 15 2017 03:41:20 "
>
  2 days ago
</time>
`;

exports[`TimeSince renders default 1`] = `
<time
  datetime="2017-10-15T01:41:20.000Z"
  title="October 15, 2017 3:41 AM "
>
  2 days ago
</time>
`;
